StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1600ElementsMustBeDocumented.cs

```diff
@@ -116,6 +121,10 @@ public void HandleMethodDeclaration(SyntaxNodeAnalysisContext context)
                 }
 
                 MethodDeclarationSyntax declaration = (MethodDeclarationSyntax)context.Node;
+                if (!(declaration.Parent is BaseTypeDeclarationSyntax))
```

Comment:
I had a quick look at the code and thought that `AccessLevelHelper` might be a better place to make this change. It has several similar methods, with only two cases where the try-cast pattern isn't used (and therefore where we might get this error). It's the two overloads of `GetEffectiveAccessibility()` for property and method definitions. We should probably fix properties too.